specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -60,7 +60,7 @@
           "description": "Gets or sets a Boolean value that indicates true if the parameter is dynamic."
         },
         "position": {
-          "type": "boolean",
+          "type": "integer",
```

Comment:
This is a breaking change for code gen in most or all languages, and this swagger is in the `stable` folder.

Reply:
Our SDK is still in preview mode and open for breaking changes (https://www.nuget.org/packages/Microsoft.Azure.Management.Automation/3.0.1-preview). Also, from what I understood, the service returns integer and not boolean, which is why we corrected this and changed to integer type from boolean. 

@avkaur Could you please confirm.